bootstrap/sql/migrations/native/1.3.0/mysql/schemaChanges.sql

```diff
@@ -32,8 +32,14 @@ set json = JSON_INSERT(
 )
 where name = 'DataInsightsApplication';
 
+-- Remove mssql connection from airflow db
+UPDATE pipeline_service_entity pse 
+SET json = JSON_REMOVE(json, '$.connection.config.connection')
```

Comment:
will this break the service? Or do we need to specify a "Backend" connection meanwhile or something 🤔

Reply:
I checked it, it does not break the service. The connection shows blank in the UI and then the same can be updated via UI